### PR TITLE
doc: releases: Release notes for v2.3 - initial (empty) draft

### PR DESCRIPTION
**Pull request by galak**
Getting things prepared for the Zephyr 2.3 release cycle

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

### CODE REVIEW

**Comment by galak** [comment]
Would like to see us merge this for 2.2 so that any PRs aimed for 2.3 can update this if needed.

**Comment by jhedberg** [comment]
> Would like to see us merge this for 2.2 so that any PRs aimed for 2.3 can update this if needed.

@galak did you mean to set the 2.2 milestone in that case? (now it's 2.3)

Btw, does this mean you're mentally preparing to volunteer to be the next release manager? ;)

**Comment by galak** [comment]
> @galak did you mean to set the 2.2 milestone in that case? (now it's 2.3)

oops was meant to be 2.2 

> Btw, does this mean you're mentally preparing to volunteer to be the next release manager? ;)

I've thought of volunteer, i just dont think I have the time right now.

**Comment by jhedberg** [comment]
@galak I'm little bit hesitant with merging this so early. In terms of a continuous git history, for someone just using the master branch it's not that remarkable, but for someone who picks the 2.2 tarball or tag it might log odd to see a release-notes-2.3.rst in the tree (to the extent that people might be asking themselves the question: did I pick the right tarball?).

**Review by carlescufi — APPROVED** [review]

**Comment by ioannisg** [comment]
Let's wait until 2.2 is out.

File: doc/releases/release-notes-2.3.rst [hunk 1]
```diff
@@ -0,0 +1,260 @@
+:orphan:
+
+.. _zephyr_2.3:
+
+Zephyr 2.3.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr kernel version 2.3.0.
```
Comment (ioannisg) [inline doc/releases/release-notes-2.3.rst]: Change kernel -> RTOS

**Review by ioannisg — DISMISSED** [review]

**Review by nashif — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**